src/System.Collections/tests/Generic/SortedList/SortedList.Tests.cs

```diff
@@ -43,6 +43,7 @@ public void IDictionary_NonGeneric_ItemSet_NullValueWhenDefaultValueIsNonNull(in
 
         [Theory]
         [MemberData(nameof(ValidCollectionSizes))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Net46, "11324")]
```

Comment:
I expect these differences to be resolved in the next NetFramework release, so it should start failing when we have the next framework runs..